frontend/@mf-types/@mf/modelRegistry/compiled-types/src/__tests__/unit/testUtils/hooks.d.ts

```
@@ -0,0 +1,79 @@
+import { RenderHookOptions, RenderHookResult, waitForOptions } from '@testing-library/react';
+import { queries, Queries } from '@testing-library/dom';
+export type BooleanValues<T> = T extends boolean | number | null | undefined | Function ? boolean | undefined : boolean | undefined | {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)